osgi/integrationtest/src/test/java/org/hibernate/validator/osgi/integrationtest/PaxExamOptions.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Hibernate Validator, declare and validate application constraints
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+package org.hibernate.validator.osgi.integrationtest;
+
+import org.ops4j.pax.exam.CoreOptions;
+import org.ops4j.pax.exam.Option;
+import org.ops4j.pax.exam.options.DefaultCompositeOption;
+
+public enum PaxExamOptions {
+	JAVA_9(
+		CoreOptions.vmOptions(
+			"--add-opens",
+			"java.base/java.security=ALL-UNNAMED",
+			"--add-opens",
+			"java.base/java.net=ALL-UNNAMED",
+			"--add-opens",
+			"java.base/java.lang=ALL-UNNAMED",
+			"--add-opens",
+			"java.base/java.util=ALL-UNNAMED",
+			"--add-exports=java.base/sun.net.www.protocol.http=ALL-UNNAMED",
+			"--add-exports=java.base/sun.net.www.protocol.https=ALL-UNNAMED",
+			"--add-exports=java.xml.bind/com.sun.xml.internal.bind.v2.runtime=ALL-UNNAMED",
+			"--add-exports=jdk.xml.dom/org.w3c.dom.html=ALL-UNNAMED",
+			"--add-exports=jdk.naming.rmi/com.sun.jndi.url.rmi=ALL-UNNAMED",
+			"--add-exports=java.xml.ws/com.sun.xml.internal.messaging.saaj.soap.impl=ALL-UNNAMED",
+			"--add-modules",
+			"java.xml.ws.annotation,java.corba,java.transaction,java.xml.bind,java.xml.ws,jdk.xml.bind" )
```

Comment:
I have 2 questions here:
- with the version of Pax-Exam we use, it's the responsibility of Pax-Exam to add these options. Has it changed?
- the java.xml.bind module will be removed with JDK 11

Do you know if there's any progress towards JDK 11 support?

Reply:
Hi @gsmet, it's no [longer](https://github.com/ops4j/org.ops4j.pax.exam2/commit/2d406793decdcdb43434ffaac34f8f6ce031866d) Pax Exam's responsibility. I think this was done so that Pax Exam doesn't fail if a module gets removed in a future release.

We will probably have to create a JAVA_11 Pax Exam option to support Java 11.

Note that Karaf doesn't work with Java 11 yet. [KARAF-5764](https://issues.apache.org/jira/browse/KARAF-5764) (In progress)

Thanks!

Reply:
OK, that makes sense.

Thanks a lot for the PR. Looking forward to the Pax-Exam release. Do you know when they plan to do it?

Reply:
Hi @gsmet, we'll try to one next week

Reply:
Nice, thanks for the heads up!